gips/0080-optimistic-ebo.md

```diff
@@ -0,0 +1,237 @@
+    - An honest proposer will be rewarded more for the won resolution than the original update.
+    - Disputers are incentivized to post the new answer, as they have already computed the query.
+- The payment from the requester will end up going to the proposer with the correct answer
+
+The “overtake” mechanism improves resolution time and reduces Arbitrator calls, while the second answer (before resolution) improves the answering time to the request. 
+
+![EBO Designs (6).png](../assets/gip-0080/Optimistic%20EBO%20Designs%20(1).png)
+
+**Source of Truth**
+
+If the escalating bond mechanism cannot resolve a dispute, the Arbitrator will be the final instance and ultimate source of truth, though this could change in the future.
+
+There are other possible sources of truth for the protocol to verify the timestamp of a block on-chain:
+
+- **Canonical bridges:** This is the most secure option since it shares the chain’s security, but on optimistic chains, it can introduce significant delays due to the mainnet withdrawal period
+- **Third-party bridges:** Less secure but varying in their levels of trust.
+    - **Light client-based bridges:** These use light clients to update block headers with timestamps, but no battle-tested solutions exist yet.
+    - **Bridge aggregators:** Combine results from multiple bridges, like [Hashi](https://github.com/gnosis/hashi) or [LayerZero](https://docs.layerzero.network/v2/home/modular-security/security-stack-dvns) for EVMs, although it requires additional development
+    - Other bridges require case-by-case analysis.
+
+> Using an Arbitrator (a Council of qualified individuals selected by the governance) is the simplest way to launch the system at this early stage and manage all the chains with a single resolution mechanism. This approach allows for greater decentralization in the future, as the source of truth can be easily migrated.
+> 
+
+**Some considerations:**
+
+- Implementing multiple resolution systems by chain or type (rollups, L1s, etc) adds complexity.
+- The Council should upgrade the system to accommodate future improvements.
+- Disputes should be managed per-chain to prevent a single dispute from impacting the entire system.
+
+## **Economic Security considerations**
+
+We recognize that economic security in The Graph presents complex challenges, which various teams are researching. The ideas discussed here are preliminary and may adapt to future protocol developments.
+
+### Parameters
+
+We suggest the following parameter values for the Escalation Bond system:
+
+- **Bond threshold**: The amount of stake that triggers a call to the Arbitrator (virtual bond in the escalation system), set at 100K GRT.
+- **Round**: The maximum number of rounds a dispute can escalate, set at 5.
+- **Min Bond**: Minimum amount required for five rounds of escalations to reach the Bond threshold, set at 11111 GRT.
```

Comment:
It's a little confusing and I'm rewriting this section.

MinBond is the amount you need to initiate a dispute. Then the proposers must overtake the minbond by doubling the bond until the proposer or disputer stops answering or the total amount bet exceeds the bond threshold and calls the arbitrator. In this case it would look like this:

Round 0 - (start dispute): 11111
Round 1- proposer 22222
Round 2- disputer 22222
Round 3- proposer 22222
Round 4- disputer 22222
Round 5- proposer 22222 -> call arbitrator

Reply:
@RembrandtK The bond gets increased by 1 bond unit (configured in the request) for every escalation. So if we set 11111 GRT as the bond, every escalation increases it by that amount